src/__stories__/config.js

```diff
@@ -0,0 +1,8 @@
+const { configure } = require("@kadira/storybook")
+
+function loadStories() {
+  require("../components/__stories__/artwork")
+  require("../components/__stories__/artwork_grid")
+}
+
+configure(loadStories, module);
```

Comment:
👍 

Could it be worth dynamically `require`ing anything that has the `__stories__` as it's a bit less intuitive on where the default is now?